components/org.wso2.carbon.identity.api.server.action.management/org.wso2.carbon.identity.api.server.action.management.v1/src/main/java/org/wso2/carbon/identity/api/server/action/management/v1/constants/ActionMgtEndpointConstants.java

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.api.server.action.management.v1.constants;
+
+/**
+ * Constants related to Action management.
+ */
+public class ActionMgtEndpointConstants {
+
+    private ActionMgtEndpointConstants() {
+    }
+
+    public static final String ACTION_MANAGEMENT_PREFIX = "ACTION-";
```

Comment:
Isn't this an internal detail used within this class. I think we can make this private

Reply:
This is used in getCode() of `ErrorMessages` enm in `ActionMgtEndpointConstants` class